CHANGELOG.md

```diff
@@ -1,5 +1,295 @@
+
+In action options, X and Y offsets that define the point where action is performed can now be negative.
+In this instance, the cursor position is calculated from the bottom-right corner of the target element.
+
+```js
+await t.click('#element', { offsetX: -10, offsetY: -30 });
+```
+
+#### Client functions as an assertion's actual value ([#1009](https://github.com/DevExpress/testcafe/issues/1009))
+
+You can now pass client functions to assertion's `expect` method. In this instance, the
+[Smart Assertion Query Mechanism](https://devexpress.github.io/testcafe/documentation/test-api/assertions/#smart-assertion-query-mechanism)
+will run this client function and use the return value as the assertion's actual value.
+
+```js
+import { ClientFunction } from 'testcafe';
+
+const windowLocation = ClientFunction(() => window.location);
+
+fixture `My Fixture`
+    .page `http://www.example.com`;
+
+test('My Test', asynct t => {
+    await t.expect(windowLocation).to.eql('http://www.example.com');
+});
+```
+
+#### Automatic waiting for scripts added during a test action ([#1072](https://github.com/DevExpress/testcafe/issues/1072))
+
+If a test action adds scripts on a page, TestCafe now automatically waits for them to finish before proceeding to the next test action.
+
+#### New ESLint plugin ([#1083](https://github.com/DevExpress/testcafe/issues/1083))
+
+We have prepared an [ESLint plugin](https://github.com/miherlosev/eslint-plugin-testcafe).
+Get it to ensure that ESLint does not fail on TestCafe test code.
+
+### Bug Fixes
+
+* Remote browser connection timeout has been increased ([#1078](https://github.com/DevExpress/testcafe/issues/1078))
+* You can now run tests located in directories with a large number of files ([#1090](https://github.com/DevExpress/testcafe/issues/1090))
+* Key identifiers for all keys are now passed to key events ([#1079](https://github.com/DevExpress/testcafe/issues/1079))
+* Touch events are no longer emulated for touch monitors ([#984](https://github.com/DevExpress/testcafe/issues/984))
+* v8 flags can now be passed to Node.js when using TestCafe from the command line ([#1006](https://github.com/DevExpress/testcafe/issues/1006))
+* ShadowUI root is now hidden for `elementFromPoint` in an iframe in IE ([#1029](https://github.com/DevExpress/testcafe/issues/1029))
+* Preventing the form submit event no longer leads to additional delay between actions ([#1115](https://github.com/DevExpress/testcafe/issues/1115))
+* TestCafe no longer hangs when a cursor is moved out of a reloading iframe ([#1140](https://github.com/DevExpress/testcafe/issues/1140))
+* Onclick event handler is now executed correctly during click automation in specific cases ([#1138](https://github.com/DevExpress/testcafe/issues/1138))
+* The `application/pdf` mime type is no longer recognized as a page ([#1014](https://github.com/DevExpress/testcafe-hammerhead/issues/1014))
```

Comment:
replace please
[#1014](https://github.com/DevExpress/testcafe-hammerhead/issues/1014)
to
[testcafe-hammerhead#1014](https://github.com/DevExpress/testcafe-hammerhead/issues/1014)

like in the v 0.11.0 changes